libs/openFrameworks/graphics/ofTrueTypeFont.cpp

```diff
@@ -600,10 +600,10 @@ bool ofTrueTypeFont::load(string _filename, int _fontSize, bool _bAntiAliased, b
 			// 1-bit format, hella funky
 			// here we unpack it:
 			unsigned char *src =  bitmap.buffer;
-			for(unsigned int j=0; j <bitmap.rows;j++) {
```

Comment:
i have unsigned int in linux, a possible solution that would work on every version would be to use:

```
for(decltype(bitmap.rows) j=0; j <bitmap.rows;j++) {
```